src/index.html

```diff
@@ -10,8 +10,171 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
+    <form method="post" action="https://mate-academy-form-lesson.herokuapp.com/   create-application"
```

Comment:
![image](https://user-images.githubusercontent.com/54936044/227705783-17ca28b5-659a-4b60-bc2f-37f8d8e5e6ab.png)
Please check all your code according to checklist